test/ui-test/testSuites/global_shared/steps/steps.py

```diff
@@ -69,7 +69,7 @@ def step(context, text, obj):
 
 @When("the user clicks on the following ui-component \"|any|\"")
 def step(context: any, obj: str):
-    the_user_clicks_on_the_following_ui_component(obj)
+    StartupSteps.the_user_clicks_on_the_following_ui_component(obj)
```

Comment:
I think it should be like this: 
 
```suggestion
    init_steps.the_user_clicks_on_the_following_ui_component(obj)
```